Assets/Code/Scripts/PlayerManagement/CameraManager.cs

```diff
@@ -8,57 +8,54 @@ public class CameraManager : MonoBehaviour
     
     [SerializeField] private GameObject _realPlayer;
     [SerializeField] private GameObject _realPlayerCamera;
-    private RealityMovement _realPlayerMovement;
+    private RealityMovement _realPlayerMovement;  // this isn't used
```

Comment:
I would remove it if it is not needed. I would keep it only if you have in mind to use it soon...